### PR TITLE
Organize Wind River Linux profiles

### PR DESCRIPTION
**Pull request by radzy**
Reorganize the Wind River Linux profiles.  The new profile
structure is that the basic/standard profile is named
"basic-embedded".  Additional profiles will be created, tailored
to specific configurations of Wind River Linux.  For example,
a profile for the Wind River Linux Security Profile configuration
would be named "wrlinux-scp".

Added support for additional rules to fill out the basic-embedded
profile better.

Signed-off-by: T.O. Radzy Radzykewycz <radzy@windriver.com>

### CODE REVIEW

File: WRLinux/input/guide.xslt [hunk 1]
```diff
@@ -16,8 +16,7 @@
       <xsl:copy-of select="@*|node()" />
 
       <!-- Adding profiles here -->
-      <xsl:apply-templates select="document('profiles/standard.xml')" />
-      <xsl:apply-templates select="document('profiles/common.xml')" />
```
Comment (mpreisler) [inline WRLinux/input/guide.xslt]: could you please also delete those files?

**Comment by redhatrises** [comment]
@radzy please remove all `cce` and `stigid` references from this PR as they are Red Hat specific.

**Comment by radzy** [comment]
@redhatrises Thanks.  My mistake.  I'm correcting it now.

File: WRLinux/input/guide.xslt (hunk 1 above)
Comment (radzy) [inline WRLinux/input/guide.xslt]: @mpreisler Thanks!  They're deleted.

**Comment by mpreisler** [comment]
@radzy Just FYI, there is no need to close PRs and open new ones. You can always change whatever you want in your local branch and `git push --force origin feature_branch`. This will show up in the PR.

So if you need to squash commits you can do it via `git rebase -i`, if you need to add more commits you can just commit and push.

**Comment by radzy** [comment]
Hmm.  Looks like I need to send a new PR instead of continuing with this one.

**Comment by mpreisler** [comment]
@radzy to be honest I don't care either way. It's just that deleting the branch and reopening a pull request wastes your time.

**Pull request closed without merging**